src/cc/api/BPF.cc

```diff
@@ -859,11 +859,12 @@ namespace {
 std::string random_alnum_string(int len) {
   static constexpr char kDict[] = "0123456789abcdefghijklmnopqrstuvwxyz";
   static std::random_device rd;
-  std::uniform_int_distribution<size_t> dist(0, sizeof(kDict)-1);
+  static std::mt19937 gen(rd());
```

Comment:
Do we really need a random device for the seed, or can we just use a fixed seed? I think all we need is uniqueness, not true run-to-run randomness.

Reply:
Done in  [2fa33ab](https://github.com/pixie-io/bcc/pull/4/commits/2fa33abf0ce0363bb121c35dceb9831c3c2c1dd1).